engine/runtime/src/main/scala/org/enso/interpreter/runtime/SerializationManager.scala

```diff
@@ -258,7 +186,7 @@ final class SerializationManager(private val context: TruffleCompilerContext) {
 
```

Comment:
The above line is:
```scala
doSerializeLibrarySuggestions(
        compiler,
        libraryName,
        useGlobalCacheLocations
)
```
that means the _serialization of suggestions_ is _piggy backing_ on serialization of library `.bindings` invoked via `doSerializeLibrary`? CCing @4e6 - I'd expect the _suggestions caches_ to be their own cache with its own life cycle, but apparently that's not the case. A slight complication for moving suggestion into `runtime-suggestions` module as suggested by
- #8894

It be good to **make the suggestions cache** a cache on its own. The question is: where to trigger its serialization?

Reply:
> It be good to make the suggestions cache a cache on its own. The question is: where to trigger its serialization?

I believe we build it during compilation (similarly to IR cache) to save time when running Enso.

Reply:
Generating the cache while we `--compile` the project is OK. I just wanted to make it an independent cache - not associated with `.bindings` file generation. But then there is this problem: https://github.com/enso-org/enso/pull/8924#issuecomment-1931277476